storage/factory/dbConfigHandler.go

```diff
@@ -75,29 +75,21 @@ func (dh *dbConfigHandler) GetDBConfig(path string) (*config.DBConfig, error) {
 
 // SaveDBConfigToFilePath will save the provided db config to specified path
 func (dh *dbConfigHandler) SaveDBConfigToFilePath(path string, dbConfig *config.DBConfig) error {
-	_, err := os.Stat(path)
```

Comment:
please put back this code. You can end up writing directories for the memory DB as you rely only on the implementation of SaveTomlFile which can be changed

Reply:
reverted to add check for directory